samples/charts/tree-map/layout-configuration/App.razor

```diff
@@ -0,0 +1,103 @@
+@using IgniteUI.Blazor.Controls
+@inject IIgniteUIBlazor IgniteUIBlazor
+
+    <div class="igContainer">
+        <div class="igOptions">
+            <div class="igOptions-horizontal">
+                <button class="igOptions-button" @onclick="onClickSliceVer" style="width: 150px">Slice Vertically</button>
+                <button class="igOptions-button" @onclick="onClickStripVer" style="width: 175px">Stripped Vertically</button>
+                <button class="igOptions-button" @onclick="onClickSquarified" style="width: 125px">Squarified</button>
+            </div>
+            <div class="igOptions-horizontal">
+                <button class="igOptions-button" @onclick="onClickSliceHor" style="width: 150px">Slice Horizontally</button>
+                <button class="igOptions-button" @onclick="onClickStripHor" style="width: 175px">Stripped Horizontally</button>
+                @*this label displays info about currently selected item in Treemap*@
+                <label>@SelectedItemInfo</label>
+            </div>
+        </div>
+
+        <div class="igComponent">
+
+            <Treemap Height="100%" Width="100%"
+                     DataSource="@Data"
+                     ParentIdMemberPath="Parent"
+                     IdMemberPath="Name"
+                     LabelMemberPath="Name"
+                     ValueMemberPath="Pop"
+                     LayoutType="@TreeLayout"
+                     LayoutOrientation="@TreeOrientation"
+                     FillScaleMode="@TreemapFillScaleMode.Value"
+                     FillScaleMinimumValue="0"
+                     FillScaleMaximumValue="1500000000"
+                     FillBrushes="#4e62cf, #8a58d6"
+                     IsFillScaleLogarithmic="true"
```

Comment:
try removing min/max FillScale props and FillScaleMode since they are covered in other sample

Reply:
pending